crates/primitives/src/alloy_compat.rs

```diff
@@ -8,6 +8,12 @@ use crate::{
 use alloy_primitives::TxKind;
 use alloy_rlp::Error as RlpError;
 
+#[cfg(feature = "optimism")]
+use crate::{
+    transaction::TxDeposit,
+    constants::OP_SYSTEM_TX_FROM_ADDR,
+};
```

Comment:
it'd like to use full paths and not feature gate imports

Reply:
confirmed.